packages/manager/apps/pci-vouchers/src/pages/ListingPage.tsx

```diff
@@ -34,6 +32,10 @@ import {
 
 import { useNavigation } from '@ovh-ux/manager-react-shell-client';
 
+import {
+  isDiscoveryProject,
```

Comment:
`isDiscoveryProject` seems unused in this file

Reply:
it's used, just check the entire file